extensions/amp-bind/0.1/bind-validator.js

```diff
@@ -159,16 +159,21 @@ export class BindValidator {
    * @private
    */
   isUrlValid_(url, rules) {
-    // @see validator/engine/validator.ParsedUrlSpec.validateUrlAndProtocol()
-    const {allowedProtocols} = rules;
-    if (allowedProtocols && url) {
-      const re = /^([^:\/?#.]+):[\s\S]*$/;
-      const match = re.exec(url);
-      if (match !== null) {
-        const protocol = match[1].toLowerCase().trim();
-        // hasOwnProperty() needed since nested objects are not prototype-less.
-        if (!hasOwn(allowedProtocols, protocol)) {
-          return false;
+    // @see validator/engine/validator.js#validateUrlAndProtocol()
```

Comment:
Browsers ignore tab/CR/LF (ie: \t\r\n) in URLs, so we should probably remove those from the URL as a preprocessing step if it's not already done elsewhere.

Reply:
Thanks, will address in follow-up.